### PR TITLE
Fixed the typos in ApertiumLintBear

### PR DESCRIPTION
**Pull request by gokberksahin**
Changed the word `consits` -> `consists` and `Inconsitency` -> `Inconsistency`.

### CODE REVIEW

**Comment by mridubhatnagar** [comment]
In the commit body of ApertiumLintBear. There is no need to mention again
the file name. "in ApertiumLintBear". You have made changes in the ApertiumLintBear is clear by the 
first line of the commit message itself.

**Comment by mridubhatnagar** [comment]
It shows 4 files changed. Though the change relevant to this issue is only in 1 file. Other files should not come up.  Why did you change the commit message body? Except for mentioning "in ApertiumLintBear" everything was fine. Make commit message proper.

**Comment by mridubhatnagar** [comment]
Changes look good to me. 
Once the travis build passes. It can be approved.

**Comment by gokberksahin** [comment]
Why the travis build has failed?

**Comment by mridubhatnagar** [comment]
Please keep it open. There is no need for it to be closed.

**Comment by mridubhatnagar** [comment]
All builds passing.
ack  b2ee597

**Comment by gokberksahin** [comment]
I did it to rebuild since travis failed first time.

**Comment by gokberksahin** [comment]
Why this PR has still process/wip label?

**Comment by mridubhatnagar** [comment]
It has process approved label. It is written process/pending review label has been removed.

**Comment by gokberksahin** [comment]
Will this be merged?

**Comment by Makman2** [comment]
Could you rephrase the shortlog to `ApertiumLintBear: Fix typos`? In the old variant you've technically no verb^^
Optional: You can actually remove the commit body (except the issue reference), as the described changes are immediately reflected in the diff.

**Comment by Makman2** [comment]
unack b2ee597

**Comment by mridubhatnagar** [comment]
Ah.. good catch @Makman2. 
I missed out on that "for". Fix Typos looks better.

**Comment by gokberksahin** [comment]
Why Circle CI is failing?

**Comment by newbazz** [comment]
You need to rebase.

**Comment by shreyateeza** [comment]
@gokberksahin The body in the commit is missing. Use git commit --amend to add the body.

**Comment by Makman2** [comment]
For trivial things a commit body is not required, especially the linked issue should really clear up last doubts :)

But yes, rebase required, we had a recent master breakage (upstream updates).

**Comment by gokberksahin** [comment]
My local repo got deleted please see: https://github.com/coala/coala-bears/pull/2192

**Comment by Makman2** [comment]
But even then you don't have to close. Please take care next time fixing your existing PR :+1: There's nearly never the necessity to close a PR^^
